### PR TITLE
Portfolio: desktop version

### PR DESCRIPTION
**Pull request by jerymey**
**In this project, I did the following:** 

- [ ] Edit HTML and CSS files
- [ ] Used correct GitHub flow
- [ ] Fixed linter errors
- [ ] Added media query for the desktop version

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -14,8 +14,6 @@
         <nav class="navbar">
             <div class="logo">
               <a href="#">My Logo</a>
-            </div>
-            <div class="menu-icon">
```
Comment (TadesseDev) [inline index.html:18]: - [ ] Dear @jerymey, Your navbar navigation items are hidden from the screen for screen sizes from `768px` to `1400px`, Please make sure we can always see the navigation items on the screen and the layout stays the same. 
- [ ] Also the logo is in multi line text, This is different from the original design. Please take a look and make some changes.
### Screen size >768px 
image

File: index.html [hunk 2]
```diff
@@ -64,84 +65,109 @@ <h4>CANOPY</h4>
                 <li class="skill-1">Javascript</li>
             </ul>
             <button class="btn"><a href="">See Project</a></button>
+    </div>   
         </article>
```
Comment (TadesseDev) [inline index.html:69]: - [ ] Dear @jerymey, For the buttons, when I hover over the button, I notice the color for the background and text becomes the same, which prevents me from reading the text while hovering over it. Please use the button interaction layout from figma and apply the sma effect.

### Your button on hover 
image

### Figma layout 
image

File: index.html [hunk 3]
```diff
@@ -152,7 +178,9 @@ <h2 class="title-card">About Myself</h2>
            <li><a href=""><img src="images/social-media-icon/github.png" alt="github"></a></li>
            <li><a href=""><img src="images/social-media-icon/v.png" alt="v"></a></li>
         </ul>
-        <button class="btn"><a href="">Get my resume</a></button>
+        <button class="btn btn-1"><a href="">Get my resume</a></button>
+    </div>
```
Comment (TadesseDev) [inline index.html:182]: - [ ] For the `About Myself` section, Please notice the left one is expected to take 40% and the right one the remaining 60% of the page's width. In your case, I can see both use 50% of the width and this is against the provided design, Please amend this change to the project. 
- [ ] Also, I can see your design overflows for screen size from `768px` to `1400px`, and I have to scroll the overflow to see the whole content. This is not a good practice when it comes to the UI design guideline. Please make sure your design is responsive and stays inside the view port at all time.

### Your design 
image

### Figma design 
image

File: index.html [hunk 4]
```diff
@@ -193,7 +221,7 @@ <h2 class="title-footer">Contact me</h2>
           <input type="text" name="name" placeholder="Your Name" maxlength="30" required />
           <input type="email" name="email" placeholder="Your Email" required />
           <textarea name="message" rows="8" placeholder="Write your message here . . ." maxlength="500" required></textarea>
-          <button type="submit" class="btn">Get in touch</button>
+          <button type="submit" class="btn btn-1">Get in touch</button>
         </fieldset>
         </form>
         </footer>
```
Comment (TadesseDev) [inline index.html:227]: - [ ] For the contact form, Please make sure all the elements of the form are centered on the page. I noticed your design is centered on some screen sizes and not on others. in order to clone the design we need to make sure all the content are centered on all the desktop screen sizes.

### Your design 
image

**Review by TadesseDev — CHANGES_REQUESTED** [review]
### Hi @jerymey 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
212094426-b41dd457-c9db-4ab3-bbe9-176dd8dd08b7




## Highlights
 - [x] Good job overall following the template design ✅

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 5]
```diff
@@ -63,85 +65,110 @@ <h4>CANOPY</h4>
...
             </ul>
-            <button class="btn"><a href="">See Project</a></button>
+            <button class="btn">See Project</button>
+    </div>
         </article>
       </section>
 
       <!-- </* ----------- ABOUT MYSELF ----------- */> -->
 
       <Section class="about-myself">
```
Comment (Alfredbis29) [inline index.html:169]: - [OPTIONAL ] I can see that you made a great design for this project but there are something little things you should work on for your project to look more professional, Compare your design to the Figma one, the arrows should have space between your languages' cards, kindly fix this for your project to match the Figma design. check this :point_down: 
Screenshot from 2023-07-14 17-01-48
Figma Design 
 image

**Review by Alfredbis29 — APPROVED** [review]
# STATUS: APPROVED   
 ## Hi  @jerymey 
Your project is complete! There is nothing else to say other than... Great job clap Congratulations! :tada:
 
 TO HIGHLIGHT:
 
 * Your project is professional :white_check_mark:
 * Good Job :+1: :smile: 
 
 Now you can merge this branch and move on 1st_place_medal: HAPPY CODING AND GOOD LUCK WITH THE COMING ONES :crossed_fingers: :


 
 ## Optional suggestions

- [x] Check Under The Review

 _Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
 
Cheers and Happy coding! :clap:  :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me @Alfredbis29 in your question so I can receive the notification.
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**